Add legal notice (impressum) and privacy policy (#16)
heartofclojure/heartofclojure: privacy-policy.md

The controller is Gaiwan GmbH, c/o Arne Brasseur, Kinzigstr. 26, 10247 Berlin/Germany (“we/us/our”).

In general Gaiwan GmbH aims to only store, process, or share the minimum information necessary. The less we know about you, the better. We do need to know certain things in order to provide services, communicate with you, or meet legal requirements. In this document you can find a complete overview of what we collect and store.

## Information gathering and usage

This privacy policy pertains to all online assets and services that are part of the official Heart of Clojure conference organization. These are:

- https://heartofclojure.eu ("the website")
- https://cfp.heartofclojure.eu ("the CFP app")
- https://activities.heartofclojure.eu ("the activities app")

We also make use of third party services to collect, store, and process information. These are:

- [MailChimp](https://mailchimp.com)

> - [Tito](https://ti.to)

Reviewer: Google is missing here. We use it for some stuff. I think so far not for user-submitted data but that may change, especially with additional forms to collect information from speakers.

Author: Given that we're not using this right now I also can't describe how we're using it. I'm also not sure at this point what we would do with it re. personal data, so I'm gonna leave this out. We can update the privacy policy if necessary.

The following sections describe the data gathered for each service, and how it is used.

### https://heartofclojure.eu ("the website")

> We do not store any information about site visitors, we do not log HTTP requests, and do not store cookies in the user's browser, tracking or otherwise. No third party cookies are stored.

Reviewer: ```suggestion
We do not store any information about site visitors, we do not log HTTP requests, and do not store cookies in the user's browser, tracking or otherwise. No third party cookies are stored. The Heart of Clojure website is hosted by Github, which may store some request information.
```
Maybe something like this to make it more clear that GitHub may store information?

When a user submits the form to sign up to our mailinglist, then this information is submitted to, and stored in MailChimp, as described below.

> The Heart of Clojure website is hosted by Github.

Reviewer: ```suggestion
```
In accordance to the change suggestion above.

### https://cfp.heartofclojure.eu ("the CFP app")

When a user submits a talk proposal using the CFP app then we ask for and store: name, email address, social media profiles, location. This data is stored on Heroku ([Heroku security information](https://www.heroku.com/policy/security) and [privacy policy](https://www.salesforce.com/company/privacy/)).

When a user uses social logins (Twitter or Github OAuth, henceforth "login provider") then a subset of their profile information at the login provider is made accessible to the CFP app and stored. Refer to the consent screen provided by the login provider for details.

This data is shared with our team of reviewers and with the program committee in order to put together a conference program. We only keep this information as long as necessary to complete this task. The CFP app instance including any stored data is destroyed one month after the conference is over, or sooner.

> Email addresses of submitters may be imported into MailChimp for the express purpose of keeping people informed on the status of the CFP process. These are kept in separate email lists, and deleted when the CFP process is completed. We do not sign people up to our main announcement list without explicit consent to do so.

Reviewer: ```suggestion
Email addresses of submitters may be imported into MailChimp for the express purpose of keeping people informed on the status of the CFP process. These are kept in separate email lists, and deleted when the CFP process is completed. We do not sign people up to our newsletter without explicit consent to do so.
```
Other lists are not newsletters and this feels clearer?

We preserve the right to share personal information of ticket holders with third parties to comply with regulations. This may include insurance providers, providers of accounting and tax related services, and government agencies in Germany and Belgium.

## Your Data

Gaiwan GmbH uses third-party vendors and hosting partners to provide the necessary hardware, software, networking, storage, and related technology required to run Heart of Clojure. You retain all rights to your data; we will never share your personal data with a third-party without your prior authorization, and we will never sell data to third-parties. We transfer data with third-parties necessary to our ability to provide our services, all of whom are GDPR-compliant and provide the necessary safeguards required if they are outside of the EU.

## Your rights

For further information on your rights, to withdraw consent, or to ask for deletion of your data, you can contact us at

> arne.brasseur@gmail.com.

Reviewer: Linebreak intentional?

Furthermore, in exercising your right to data portability, you have the right to have personal data transmitted directly from one controller to another where technically feasible — and when doing so does not affect the rights and freedoms of others.

### Right to withdraw consent

You have the right to withdraw your consent regarding the use, processing or transmission of your data at any time in writing or by email to us.

In the event of consent withdrawal, we will no longer process and immediately delete your stored data. This does not apply if we can prove compelling grounds for processing that are worthy of protection and which outweigh your interests, rights and freedoms or in case the processing serves to assert, exercise or defend legal claims. For example, we will continue to use data if it is still necessary for tax purposes.

## Duration of Storage

We store your data as long as necessary to organize Heart of Clojure, or as long as it is needed for legal or tax filing purposes.

> Generally we strive to delete all personal information within one month after the end of the conference.

Reviewer: Conflicting with earlier sections that state hard deletion date as 1mo after conference.

Author: Updated to be unambiguous.
